### ISSUE TITLE
Exim filter botnet noise

### ISSUE THREAD

**Issue by hungerburg** [fail2ban/fail2ban#1983]
Hello, I propose to add two new failregex' to _exim.conf_ - they spare me thousands of lines a day in mainlog, so it remains readable for debugging :) Here they are:
```
^%(pid)s no IP address found for host \S+ \(during SMTP connection from \[<HOST>\]\)$
^%(pid)s no host name found for IP address <HOST>$
```
PS: From tcpdumping the conversations I know, that they get written on SMTP protocol errors due early AUTH (there is a failregex for already in), that never make it to the logfile though.

PPS: As far as I'm concerned, if that penalizes honest smtpds on dialup, I'd tell them to get a smarthost.

Peter

**Comment by sebres** [comment]
No problem with such PR, but only as extension (similar `[mode=aggressive]` for other filters).
BTW, please provide example log-lines (excerpt) for such failures.

**Comment by hungerburg** [comment]
@sebres Even after banning after the 6th failure, I can provide lots of such log-lines ;) Below two of those:
```
2017-12-03 08:51:35 no IP address found for host 212-129-9-163.rev.poneytelecom.eu (during SMTP connection from [212.129.9.163])
2017-12-03 08:32:00 no host name found for IP address 114.99.5.147
```
Peter

**Comment by sebres** [comment]
Solved in #1988 for \>= 0.10, I'll merge it in the near future.
For v.0.9 it could be customized with the regex you suggested.

**Comment by hungerburg** [comment]
Cool glasses :) Keep up the work!
